src/components/dropdownSearch/DropdownSearch.tsx

```diff
@@ -98,41 +124,45 @@ export class DropdownSearch extends React.Component<IDropdownSearchProps, void>
           ? this.getTextFiltered(value)
           : value;
 
-        return <li key={option.value}
-          className={liClasses}
-          title={value}>
-          <span className={optionClasses}
-            onMouseDown={(e: React.MouseEvent<HTMLSpanElement>) => this.handleOnOptionClick(e)} data-value={option.value}>
-            {this.getDropdownPrepend(option)}
-            {this.getSvg(option)}
-            {valueToShow}
-          </span>
-        </li>;
+        return (
+          <li key={option.value}
+            className={liClasses}
+            title={value}>
+            <span className={optionClasses}
+              onMouseDown={(e: React.MouseEvent<HTMLSpanElement>) => this.handleOnOptionClick(e)} data-value={option.value}>
+              {this.getDropdownPrepend(option)}
+              {this.getSvg(option)}
+              {valueToShow}
+            </span>
+          </li>
+        );
       })
       .value();
 
     return options.length ? options : this.getNoOptions();
   }
 
-  private getNoOptions(): JSX.Element[] {
+  protected getNoOptions(): JSX.Element[] {
     return [
       <li key='noResultDropdownSearch'>
         <span>{this.props.noResultText}</span>
       </li>,
     ];
   }
 
-  private getSvg(option: IDropdownOption): JSX.Element {
-    if (option.svg) {
-      return <span key={option.svg.name}
-        className='value-icon'>
-        <Svg {...option.svg} />
-      </span>;
+  protected getSvg(option: IDropdownOption): JSX.Element {
+    if (option && option.svg) {
+      return (
+        <span key={option.svg.name}
```

Comment:
Curiously, I did not make any changes on this method, but it might have been moved around. I think PO made this in his original DropdownSearch